examples/contract-deployment/main.rs

```diff
@@ -0,0 +1,60 @@
+use ethers::abi::Abi;
+use std::str::FromStr;
+use zksync_web3_rs::prelude::abigen;
+use zksync_web3_rs::providers::{Middleware, Provider};
+use zksync_web3_rs::signers::{LocalWallet, Signer};
+use zksync_web3_rs::zks_wallet::DeployRequest;
+use zksync_web3_rs::ZKSWallet;
+
+// This is the default url for a local `era-test-node` instance.
+static ERA_PROVIDER_URL: &str = "http://127.0.0.1:8011";
```

Comment:
Is 8011 really the default port? On the readme it says 3050, on my machine it's running on the 3050, this makes the example fail.

Reply:
Nope, check the following comment. For this example I chose to use the brand-new [era-test-node](https://github.com/matter-labs/era-test-node) which runs faster than the full [local-setup](https://github.com/matter-labs/local-setup) network and it is much easier to run.

```
This is the default url for a local `era-test-node` instance`
```

Reply:
Also, this was manually tested, so I'm certain it works (on my machine, LOL)